pkg/infradriver/ssh_infradriver.go

```diff
@@ -81,7 +82,8 @@ func NewInfraDriver(cluster *v2.Cluster) (InfraDriver, error) {
 	var err error
 	ret := &SSHInfraDriver{
 		clusterName:       cluster.Name,
-		clusterImagerName: cluster.Spec.Image,
+		clusterImageName:  cluster.Spec.Image,
```

Comment:
It's confusing here, what's the meaning of clusterImageName and will there be `appImageName` in the future?

Reply:
yes, its a string value that just represent the name of this clusterimage to launch.